src/tests/libecalc/presentation/exporter/test_ltp.py

```diff
@@ -399,29 +403,45 @@ def test_only_venting_emitters_no_fuelconsumers():
     assert emissions_ch4 == emissions_ch4_asset
 
 
-def test_no_emitters_or_fuelconsumers():
+class EmptyResourceService(ResourceService):
+    def get_resources(self, configuration: YamlValidator) -> dict[str, Resource]:
+        return {}
+
+
+def test_no_emitters_or_fuelconsumers(
```

Comment:
This test should probably be moved somewhere else

Reply:
Agree, maybe in https://github.com/equinor/ecalc/blob/main/src/tests/libecalc/presentation/yaml/test_venting_emitter.py